src/routeGeneration/routeGenerator.ts

```diff
@@ -22,7 +22,8 @@ export class RouteGenerator {
         tsconfig: true,
         tsfmt: true,
         tslint: true,
-        verify: true
+        verify: true,
+        vscode: true
```

Comment:
This was related to the issue I was talking about in typescript-formatter.  They just expect vscode to be defined.  The default is true, which allows it to use .vscode/settings.json https://github.com/vvakame/typescript-formatter/issues/87